db/migrate/20241112061722_add_type_to_equipment.rb

```diff
@@ -0,0 +1,9 @@
+class AddTypeToEquipment < ActiveRecord::Migration[7.1]
+  def change
+    add_column :equipment, :type, :string
```

Comment:
Careful with a column called 'type' in ActiveRecord. It is reserved for polymorphic single table inheritance.
Best call it something else.

Reply:
Ha. The next migration does exactly that, all good :)